client/landing/stepper/declarative-flow/internals/components/stepper-loader/index.tsx

```diff
@@ -1,10 +1,46 @@
-import clsx from 'clsx';
-import WordPressLogo from 'calypso/components/wordpress-logo';
+import { isWooExpressFlow, isTransferringHostedSiteCreationFlow } from '@automattic/onboarding';
+import { ProgressBar } from '@wordpress/components';
+import { LoadingBar } from 'calypso/components/loading-bar';
 import './style.scss';
 
-const StepperLoader = () => {
-	/* eslint-disable wpcalypso/jsx-classname-namespace */
-	return <WordPressLogo size={ 72 } className={ clsx( 'wpcom-site__logo', 'stepper-loader' ) } />;
+interface StepperLoaderProps {
+	title?: string;
+	subtitle?: React.ReactNode;
+	progress?: number;
+	flow?: string;
+}
+
+const StepperLoader: React.FC< StepperLoaderProps > = ( {
+	title,
+	subtitle,
+	progress = -1,
+	flow = null,
+} ) => {
+	const renderProgressComponent = () => {
+		if ( isWooExpressFlow( flow ) || isTransferringHostedSiteCreationFlow( flow ) ) {
```

Comment:
What's the reason for these special cases?

Reply:
It is mostly because Woo uses different styles and I didn't want to remove it at this time.
But I did some changes and made this not useful anymore. Removed.